examples/howto/plot_record_extracellular_potentials.py

```diff
@@ -98,9 +98,10 @@
 # Add spike raster to subplot
 net.cell_response.plot_spikes_raster(ax=axs[2], show=False)
 
-# Finally, add the CSD to the bottom subplot
+# Finally, add the CSD to the bottom subplot, with the colormap ranging from -0.02 to 0.02 µV,
+# and spline interpolation for smoothing
 net.rec_arrays['shank1'][trial_idx].smooth(window_len=window_len).plot_csd(
-    ax=axs[3], show=False)
+    interpolation='spline', vmin=-0.02, vmax=0.02, ax=axs[3], show=False)
```

Comment:
perhaps we leave this out? Or break this into two lines? Looks a bit intimidating to a new python user right now

Reply:
I think this was originally my suggestion just to make users aware of these options

point is well taken though, it might be better to just show a short function that makes nice looking plot with no fuss, I'd be fine to see it removed